dev-haskell/ghc-lib-parser-ex/ghc-lib-parser-ex-8.10.0.16.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.6.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="Algorithms on GHC parse trees"
+HOMEPAGE="https://github.com/shayne-fletcher/ghc-lib-parser-ex#readme"
+SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	>=dev-haskell/uniplate-1.5:=[profile?]
+	>=dev-lang/ghc-8.2.1:=
+	=dev-haskell/ghc-lib-parser-8.10*:=[profile?]
```

Comment:
Should there be a commit adding `ghc-lib-parser-8.10`?

Reply:
I think this was added in #1082

Reply:
I don't see it in that pull.

Reply:
Hm, I guess it’s because I rebased on top of master then force pushed. If you go to the [commit history](https://github.com/ezzieyguywuf/gentoo-haskell/commits/hlint) of the branch you can see it right in front on my first commit.


Should I instead have merged master?

Reply:
I'm afk and not looking closely but I don't see it. CI failed because it couldn't find ghc-lib-parser-8.10. I've restarted the CI to check again in any case.

Reply:
I missed up the rebase. Good thing for CI!

I'll fix it when I push the emacs thing

Reply:
Ok, this should be fixed now, feel free to re-open this thread if the CI fails.